src/api/flaskr/api/sms/aliyun.py

```
@@ -83,6 +83,43 @@ def send_sms_ali(
+    config = open_api_models.Config(
+        access_key_id=app.config["ALIBABA_CLOUD_SMS_ACCESS_KEY_ID"],
+        access_key_secret=app.config["ALIBABA_CLOUD_SMS_ACCESS_KEY_SECRET"],
+    )
+    config.endpoint = "dysmsapi.aliyuncs.com"
+    client = Dysmsapi20170525Client(config)
+    request = dysmsapi_20170525_models.GetSmsTemplateRequest()
+    request.template_code = resolved_template_code
+    runtime = util_models.RuntimeOptions()
+    try:
+        return client.get_sms_template_with_options(request, runtime)
+    except Exception as error:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)